astro/src/content/docs/get-started/download-and-install/silent-mode.mdx

```diff
@@ -11,14 +11,14 @@ Silent mode has been available in FusionAuth since version `1.0.0`. This feature
 
 Silent mode serves two purposes that we will cover in the following sections: installation and upgrades. Keep in mind that both of these processes might fail so you should always be ready to intervene and you should always make backups of existing databases.
 
-## Installation
+## Behavior
 
-Silent can be used to automatically create the database and schema that FusionAuth uses. It can also grant the correct privileges to the correct database users so that FusionAuth is not connecting to the database as a super user. When enabled, silent mode will perform the following steps:
+Silent mode automatically creates everything in the database that FusionAuth needs without prompts in the browser. When enabled, silent mode will perform the following steps:
 
 1. Open a raw socket to the database on the correct port to ensure it is reachable
 2. If the configuration contains super user credentials, attempt to connect to the FusionAuth database using the super user (i.e. the `fusionauth` database and not the `mysql` or `postgres` database)
```

Comment:
Should this use of `attempt` also be updated? I'm not sure whether step 4 was updated because "attempt" was incorrect, or if it was a verbiage thing.
```suggestion
2. If the configuration contains super user credentials, connect to the FusionAuth database using the super user (i.e. the `fusionauth` database and not the `mysql` or `postgres` database)
```